ci-operator/step-registry/sandboxed-containers-operator/e2e/azure/sandboxed-containers-operator-e2e-azure-workflow.yaml

```diff
@@ -5,11 +5,15 @@ workflow:
       SLEEP_DURATION: 3h
       TEST_PARALLEL: 1
       FORCE_SUCCESS_EXIT: no
+      MUST_GATHER_IMAGE: "registry.redhat.io/openshift-sandboxed-containers/osc-must-gather-rhel9:latest"
+      MUST_GATHER_TIMEOUT: "35m"
+      ENABLE_MUST_GATHER: "true"
+      MUST_GATHER_ON_FAILURE_ONLY: "false"
```

Comment:
I'd suggest "true" here as most of the time we don't need that on success, do we?

Reply:
Fixed.  It was also missing from the aws workflow